ui/components/ui/editable-label/editable-label.js

```diff
@@ -40,48 +49,44 @@ export default class EditableLabel extends Component {
     );
 
     return (
-      <div className={classnames('editable-label', this.props.className)}>
-        <input
-          type="text"
+      <Box display={DISPLAY.FLEX} gap={3}>
+        <FormTextField
           required
-          dir="auto"
           value={this.state.value}
           onKeyPress={(event) => {
             if (event.key === 'Enter') {
               this.handleSubmit(isValidAccountName);
             }
           }}
-          onChange={(event) => this.setState({ value: event.target.value })}
+          onChange={(event) => {
+            this.setState({ value: event.target.value });
+          }}
           data-testid="editable-input"
-          className={classnames('large-input', 'editable-label__input', {
-            'editable-label__input--error': !isValidAccountName,
-          })}
+          error={!isValidAccountName}
+          helpText={errorMessage}
           autoFocus
+          placeholder={this.context.t('accountName')}
         />
         <ButtonIcon
           iconName={IconName.Check}
-          className="editable-label__icon-button"
           onClick={() => this.handleSubmit(isValidAccountName)}
         />
-        <div className="editable-label__error editable-label__error-amount">
-          {errorMessage}
-        </div>
-      </div>
+      </Box>
     );
   }
 
   renderReadonly() {
     return (
-      <div className={classnames('editable-label', this.props.className)}>
-        <div className="editable-label__value">{this.state.value}</div>
+      <Box display={DISPLAY.FLEX} alignItems={AlignItems.center} gap={3}>
```

Comment:
Let's not remove the className since the EditableLabel has been used as a child component in AccountDetailsModal as well and className has been passed as a prop. 
```suggestion
      <Box className={classnames('editable-label', this.props.className)} display={DISPLAY.FLEX} gap={3}>
```